chat-sdk/src/main/java/com/amazon/connect/chat/sdk/network/AttachmentsManager.kt

```diff
@@ -53,7 +53,9 @@ class AttachmentsManager @Inject constructor(
                 file
             ) { response ->
                 CoroutineScope(Dispatchers.IO).launch {
+                    print("DEBUG!!")
                     if (response != null && response.isSuccessful) {
+                        print("DEBUG2!!")
```

Comment:
We need to remove these print statements.

Reply:
Good catch, removed.